src/main/java/at/laborg/briss/gui/MergedPanel.java

```diff
@@ -276,8 +280,7 @@ public void setSelCropSize(int width, int height) {
     public void resizeSelCrop(int width, int height) {
         for (DrawableCropRect crop : crops) {
             if (crop.isSelected()) {
-                if (((width < 0) && (crop.width <= -width)) || ((height < 0) && (crop.height <= -height)))
-                    return;
+                if (((width < 0) && (crop.width <= -width)) || ((height < 0) && (crop.height <= -height))) return;
```

Comment:
I'm not  a big fan of putting the one line after the `if`. But I'm also not a fan of omitting the parenthesis 😄.
So maybe we should add the parenthesis one day (no need to do that in this PR) and this issue gets resolved anyway.

Reply:
You mean the curly braces around the if? Totally agreed. :D I'll see if I can include this as a rule for the formatter.